### DISCUSSION TITLE
Does lmdeploy support InternVL2?

### DISCUSSION THREAD

**Discussion by nickmitchko** [InternLM/lmdeploy#2002] [General]
Simple question: does lmdeploy currently support [InternVL2](https://huggingface.co/OpenGVLab/InternVL2-40B)

**Comment by nickmitchko** [comment]
Yes, it seems to:

But according to the issue [here](https://github.com/InternLM/lmdeploy/issues/1965) the 40B parameter model is causing bad output.

**Reply by lvhan028** [reply]
The latest main branch has supported InternVL2 full-series models, which will be released in next version v0.5.1 in next week
